pkg/resources/kafka/pod.go

```diff
@@ -193,6 +193,7 @@ func getInitContainers(brokerConfigInitContainers []corev1.Container, kafkaClust
 				Name:      "extensions",
 				MountPath: "/opt/kafka/libs/extensions",
 			}},
+			Resources: *brokerConfig.InitContainersResourceRequirements,
```

Comment:
what we really miss here are the sensible defaults for builtin init containers.
I suggest we provide them statically here inline (via a helper method or so).

As @stoader also correctly pointed out - any additional init container can already provide resource requirements via the `InitContainers` struct.